client/src/components/Chat/index.tsx

```diff
@@ -94,8 +94,10 @@ const Chat = () => {
               queryIdToEdit ? `&parent_query_id=${queryIdToEdit}` : ''
             }`
           : ''
-      }&model=${
-        preferredAnswerSpeed === 'normal' ? 'gpt-4' : 'gpt-3.5-turbo-finetuned'
+      }&answer_model=${
```

Comment:
Making it compatible with Answer struct interface change 'model' => 'answer_model'